### ISSUE TITLE
Generation.getGenerationById throwing errors

### ISSUE THREAD

**Issue by adaboese** [Leonardo-Interactive/leonardo-ts-sdk#76]
```ts
zodError: [
  {
    "expected": "'ANIME' | 'CINEMATIC' | 'CREATIVE' | 'DYNAMIC' | 'ENVIRONMENT' | 'GENERAL' | 'ILLUSTRATION' | 'LEONARDO' | 'NONE' | 'PHOTOGRAPHY' | 'RAYTRACED' | 'RENDER_3D' | 'SKETCH_BW' | 'SKETCH_COLOR' | 'VIBRANT'",
    "received": "null",
    "code": "invalid_type",
    "path": [
      "object",
      "generations_by_pk",
      "presetStyle"
    ],
    "message": "Expected 'ANIME' | 'CINEMATIC' | 'CREATIVE' | 'DYNAMIC' | 'ENVIRONMENT' | 'GENERAL' | 'ILLUSTRATION' | 'LEONARDO' | 'NONE' | 'PHOTOGRAPHY' | 'RAYTRACED' | 'RENDER_3D' | 'SKETCH_BW' | 'SKETCH_COLOR' | 'VIBRANT', received null"
  }
]
    at Object.get error [as error] (/Users/ada/Developer/adaboese/aimd/node_modules/.pnpm/zod@3.22.4/node_modules/zod/lib/types.js:43:31)
    at ZodEffects.parse (/Users/ada/Developer/adaboese/aimd/node_modules/.pnpm/zod@3.22.4/node_modules/zod/lib/types.js:143:22)
    at Generation.getGenerationById (/Users/ada/Developer/adaboese/aimd/node_modules/.pnpm/@leonardo-ai+sdk@3.0.0_zod@3.22.4/node_modules/@leonardo-ai/sdk/src/sdk/generation.ts:275:80)
```

Started happening after the last update.

**Comment by disintegrator** [comment]
Hi @adaboese, according to the api spec that field is marked as non-nullable and the error you shared is saying you’re passing null instead of one of the accepted values. Are you passing a non-null value and still seeing this error?

**Comment by disintegrator** [comment]
Ah apologies for misunderstanding! You’re right about the method signature. It seems that the _server response_ contains the problematic field. I’ll get back to you soon.

**Comment by nerfologist** [comment]
@disintegrator I'm also blocked by this error, any news?

**Comment by monstermax** [comment]
never mind, let's try the api curl...

**Comment by bryanmig** [comment]
@disintegrator I signed up for a paid account this week and am unable to use the SDK because of this error.  When can we expect this to be resolved?

**Comment by disintegrator** [comment]
Hi, I suspect I caused some confusion and want to apologies for it. I don't work on the Leonardo team and I don't have access to address the underlying issue. I was helping debug the issue from the outside as an interested party.

**Comment by mecsantos** [comment]
Hi @bryanmig , I work for Leonardo. We've escalated this the engineering team. I'll update you here.

**Comment by WilliamDGH-Leo** [comment]
This has been fixed now by [v4.4.7](https://github.com/Leonardo-Interactive/leonardo-ts-sdk/releases/tag/v4.4.7).

Thanks for your support and patience.
